aws/resource_aws_guardduty_member.go

```diff
@@ -37,6 +39,29 @@ func resourceAwsGuardDutyMember() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"relationship_status": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"invite": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+			"disable_email_notification": {
```

Comment:
This attribute needs documentation added 👍

Reply:
added following: 

`* `disable_email_notification` - (Optional) Specifies whether an email notification is sent to the accounts.
`